client/src/pages/MainMenu.js

```diff
@@ -34,36 +34,51 @@ const MenuWrapper = styled.div`
 
 const MainMenu = () => {
   const history = useHistory();
+  const buttons = [
+    {
+      category: "settings",
+      design: "menu",
+      label: "Einstellungen",
+      route: "/settings",
+    },
+    {
+      category: "tours",
+      design: "menu",
+      label: "Touren",
+      route: "/tours",
+    },
+    {
+      category: "riders",
+      design: "menu",
+      label: "Riders",
+      route: "/riders",
+    },
+    {
+      category: "customers",
+      design: "menu",
+      label: "Kunden",
+      route: "/customers",
+    },
+    {
+      category: "go",
+      design: "cta",
+      label: "Lets Fetz",
+      route: "/tours/today",
+    },
+  ];
```

Comment:
You can move this outside of this function to avoid recreating this constant on every render.